resources/sass/app.scss

```diff
@@ -23,7 +23,9 @@
 @import "modules/page";
 @import "modules/control";
 @import "modules/cookies-notification";
-//Module templates
+
+// Templates
+@import "modules/video-template";
```

Comment:
it would be cool if you added this into the `templates` folder. So you'd be importing `modules/templates/video.scss`